Tests/Fixtures/TestProject/xcode12_workaround.xcconfig

```diff
@@ -0,0 +1,9 @@
+//
+// See https://github.com/Carthage/Carthage/issues/3019
+//
+// Skips building ARM slices for simulators until Carthage can support it
+//
+
+EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64=arm64 arm64e armv7 armv7s armv6 armv8
+EXCLUDED_ARCHS_1200=$(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT))
+EXCLUDED_ARCHS=$(inherited) $(EXCLUDED_ARCHS_$(XCODE_VERSION_MAJOR))
```

Comment:
This will only set `EXCLUDED_ARCHS` if building with Xcode 12. It works around an issue where Carthage fails to create fat Simulator + Device binaries. For instance, creation of fat dylibs/archives fail because `iphonesimulator` builds targeting Apple silicon share the `arm*` archs with `iphoneos`. We'll probably need this workaround until Carthage supports xcframeworks or some other means of being able to build frameworks per-platform.